sisyphus/http_server.py

```diff
@@ -268,6 +269,7 @@ def visualize(block_id):
     url_prefix = '/vis/' + block_id
 
     if parent is None:
+        items += all_root_blocks[0].get_sub_blocks()
```

Comment:
Why are only sub_blocks of the first root_block added? Better would be
root_block_a
sub_block_a1
sub_block_a2
...
root_block_b
sub_block_b1
...